README.md

```diff
@@ -1,2 +1,16 @@
 # Kizuna
 Friendship Game.  For Vote, Rank,  QA ...
+
+
+### Run Project For Dev Mode
+
+```
+npm install
+npm start
+```
+
+### Run lint
+
+```
+npm run lint/eslint
```

Comment:
I suggest change this to an executable command, with copy and paste, this won't work.